src/concrete/ml/sklearn/neighbors.py

```diff
@@ -123,3 +124,26 @@ def load_dict(cls, metadata: Dict):
         obj.metric_params = metadata["metric_params"]
         obj.n_jobs = metadata["n_jobs"]
         return obj
+
+    # KNeighborsClassifier does not provide a predict_proba method for now
+    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3962
+    def predict_proba(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.ndarray:
```

Comment:
better to raise a clear error while it's not implemented, and then have a test on it (similar to NN regressor)

Reply:
thanks